dimension-reduction/README.md

```diff
@@ -0,0 +1,31 @@
+# Dimension Reduction
+
+In this module, we'll demonstrate how to use dimension reduction techniques with
+gene expression data obtained from [refine.bio](https://www.refine.bio/).
+Dimension reduction is useful for examining overall patterns of your data.
+Reducing your data to two dimensions allows you to more easily plot your samples
+so you can observe global and local patterns in your dataset.
```

Comment:
What do you mean by this?

Reply:
You can see how samples spread around overall, and you can also see how samples group (or don't group together) ie what's their neighbors, vs also overall structure.

Reply:
Can you rewrite this sentence to include that information?